db/model.js

```diff
@@ -28,19 +28,29 @@ function fetchArticles() {
       GROUP BY articles.article_id
       ORDER BY articles.created_at DESC;
       `)
-    .then(({ rows }) => {
-        return rows
-    })
+        .then(({ rows }) => {
+            return rows
+        })
 }
 
 function fetchArticleComments(article_id) {
     return db.query(`SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`, [article_id])
         .then(({ rows }) => {
-            if(rows.length === 0) {
-                return Promise.reject({ status: 404, msg: 'article does not exist' });
+            if (rows.length === 0) {
+                return Promise.reject({ status: 404, msg:'article does not exist' });
             }
             return rows;
         })
 }
 
-module.exports = { fetchTopics, fetchArticle, fetchArticles, fetchArticleComments }
\ No newline at end of file
+function insertComment(article_id, username, body ) {
```

Comment:
Nice! Consider how the additional tests might end up affecting how your endpoint works :)